src/node/cli.ts

```diff
@@ -44,6 +44,7 @@ Options:
   --jsx                      ['vue' | 'preact' | 'react']  choose jsx preset (default: 'vue')
   --jsx-factory              [string]  (default: React.createElement)
   --jsx-fragment             [string]  (default: React.Fragment)
+  --force                    [boolean] clear the optimizer cache
```

Comment:
```suggestion
  --force                    [boolean] force the optimizer to ignore the cache
```

Not exactly `clear the cache` in this context I think. Might cause some misleading.